javascript/javascript1/week3/tvTime.js

```diff
@@ -0,0 +1,32 @@
+const seriesDurations = [{
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+},
+{
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+},
+{
+    title: "The Wire",
+    days: 6,
+    hours: 17,
+    minutes: 0,
+}
+];
+let totalTime = 0;
+const totalLifeTime = 80 * 365 * 24 * 60;
```

Comment:
Since your only using these values in the function scope, you should move them in there.